### ISSUE TITLE
App is installed, no devices shown

### ISSUE THREAD

**Issue by steventoribio** [dracos/soundcaster#5]
Hello,

I have installed the app, and while other devices find the Chromecast Audio device on my network, Soundcaster apparently doesn't: when I click **_Cast to_** I only get the option _**Refresh**_, I click on it, but no devices appear.

**Comment by dracos** [comment]
Yes, the same happens for me if I use the version in the zip file. The identical (`diff -qr SC.app SC2.app` has no output) copy that I compiled in XCode works as expected. If you compile it yourself, it should work fine (as I say, it does for me), I don't know what causes the difference or what to do about it to create a working distributable version, sorry.
